nml/expression/spritegroup_ref.py

```diff
@@ -79,7 +79,8 @@ def supported_by_action2(self, raise_error):
         return True
 
     def collect_references(self):
-        return [self]
+        from itertools import chain
+        return [self] + list(chain.from_iterable(p.collect_references() for p in self.param_list))
```

Comment:
Better to have imports at the top of the file, otherwise we end up re-evaluating the import on each call.

Maybe `return list(itertools.chain(self, *(p.collect_references() for p in self.param_list)))` ?